### DISCUSSION TITLE
🍺🍷🍶What is your favorite alcoholic drink?🍹

### DISCUSSION THREAD

**Discussion by harurunrunrun** [NAIST-SE/AttractingContributors#1084] [2024 Attraction Contributions]
Please tell me drinks you like or you drinks.

https://github.com/harurunrunrun/alcoholic-drink

**Comment by gill876** [comment]
I really love Japanese plum liqueur 【梅酒】!

Please contribute to my project!
https://github.com/gill876/AttractingContributors-Hiking

**Comment by buccalcone** [comment]
I love Hakkaisan because it goes well with grilled fish.

If you can please contribute to my project!
https://github.com/YU762/AttractingContributors-Animals
